### PR TITLE
Update PPP

### PR DESCRIPTION
**Pull request by jehousoh**

### CODE REVIEW

File: docs/team/jehousoh.md [hunk 1]
```diff
@@ -5,42 +5,68 @@ title: Soh Je Hou's Project Portfolio Page
...
+  * Updated isRoomClean to a green/red icon signifying yes/no respectively.
+  * Enable text wrapping.
+  * Allow Ui to be resizable.
 
 * **Documentation**:
   * User Guide:
-    * to be added soon
+    * Added "Components of the UI" section.
+    * Added UiComponentsLabels.png.
+    * Added acknowledgements for icons.
```
Comment (bryanhce) [inline docs/team/jehousoh.md]: Should the acknowledgement be under DG instead of UG?

**Review by bryanhce — COMMENTED** [review]
1 minor change!

File: docs/team/jehousoh.md (hunk 1 above)
Comment (jehousoh) [reply docs/team/jehousoh.md]: Good catch! Changed.

**Review by bryanhce — APPROVED** [review]
LGTM

**Pull request merged**